x/leverage/spec/02_state.md

```diff
@@ -35,20 +35,22 @@ The `0x01` prefix above allows a governance-controlled `Token Registry` to be st
 
 ```go
 type Token struct {
-    BaseDenom            string
-    ReserveFactor        sdk.Dec
-    CollateralWeight     sdk.Dec
-    LiquidationThreshold sdk.Dec
-    BaseBorrowRate       sdk.Dec
-    KinkBorrowRate       sdk.Dec
-    MaxBorrowRate        sdk.Dec
-    KinkUtilization      sdk.Dec
-    LiquidationIncentive sdk.Dec
-    SymbolDenom          string
-    Exponent             uint32
-    EnableMsgSupply      bool
-    EnableMsgBorrow      bool
-    Blacklist            bool
-    MaxCollateralShare   uint32
+    BaseDenom              string
+    ReserveFactor          sdk.Dec
+    CollateralWeight       sdk.Dec
+    LiquidationThreshold   sdk.Dec
+    BaseBorrowRate         sdk.Dec
+    KinkBorrowRate         sdk.Dec
+    MaxBorrowRate          sdk.Dec
+    KinkUtilization        sdk.Dec
+    LiquidationIncentive   sdk.Dec
+    SymbolDenom            string
+    Exponent               uint32
+    EnableMsgSupply        bool
+    EnableMsgBorrow        bool
+    Blacklist              bool
+    MaxCollateralShare     sdk.Dec
+    MaxSupplyUtilization   sdk.Dec
+    MinCollateralLiquidity sdk.Dec
 }
```

Comment:
instead of copying the structure, let's create a link to generated code. This way it will be easier to keep it in sync.

Reply:
up